vllm/platforms/interface.py

```diff
@@ -330,6 +330,27 @@ def get_device_communicator_cls(cls) -> str:
         """
         return "vllm.distributed.device_communicators.base_device_communicator.DeviceCommunicatorBase"  # noqa
 
+    @classmethod
+    def supports_fp8(cls) -> bool:
+        """
+        Returns whether the current platform supports F8 types.
+        """
+        return False
+
+    @classmethod
+    def is_fp8_fnuz(cls) -> bool:
+        """
+        Returns whether the preferred F8 type is fnuz on the current platform.
```

Comment:
Maybe add a comment here about what platforms use this dtype?

Reply:
Done.  I also added links to the 2 different FP8 standards for additional clarification, and the other API `fp8_dtype` refers back to the doc for `is_fp8_fnuz` to avoid copy/paste.